docs/user/building-images.md

```diff
@@ -128,6 +122,35 @@ Build the OS image for your target platform:
 sudo podman build -t ${OCI_IMAGE_REPO}:${OCI_IMAGE_TAG} .
 ```
 
+#### Using RHEL base images
+
+When using Flight Control with a RHEL 9 base image, you need to make a few changes to the `Containerfile`, specifically you need to disable RHEL's default automatic updates and use a different command to enable the EPEL repository in case you need `podman-compose`:
+
+```console
+FROM registry.redhat.io/rhel9/rhel-bootc:9.6
```

Comment:
Is 9.6 available?  I thought the latest is 9.5?

Reply:
Ah, c**p, you're right. Thank you.